### PR TITLE
Feature/513 member profile

### PR DESCRIPTION
**Pull request by skuhlmann**
## GitHub Issue

[513](https://github.com/HausDAO/daohaus-monorepo/issues/513)

## Changes

- profile page
- makes table more re-usable

## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs locally
- [x] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

File: apps/core-app/index.html [hunk 1]
```diff
@@ -4,11 +4,11 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/src/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>DAOhaus Core</title>
+    <title>DAOhaus Admin</title>
   </head>
   <body>
     <div id="root"></div>
-      <script>
+    <script>
```
Comment (brossetti1) [inline apps/core-app/index.html:11]: off

File: apps/core-app/src/Routes.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ const Routes = () => {
         <Route path="proposal/:proposalId" element={<Proposals />} />
         <Route path="vaults" element={<Vaults />} />
         <Route path="members" element={<Members />} />
-        <Route path="member/:memberAddress" element={<Members />} />
+        <Route path="members/:memberAddress" element={<Member />} />
```
Comment (brossetti1) [inline apps/core-app/src/Routes.tsx:33]: typo - this should be `member/:memberAddress`

Comment (skuhlmann) [reply apps/core-app/src/Routes.tsx]: oh i thought in restful routing you used the plural.

Comment (brossetti1) [reply apps/core-app/src/Routes.tsx]: oh my bad - yeah you do, i think i saw members on the line above and thought it was duplicated here - ignore this

File: apps/core-app/src/pages/Home.tsx [hunk 3]
```diff
@@ -1,9 +1,12 @@
+import { useEffect } from 'react';
 import { HausLayout } from '@daohaus/daohaus-connect-feature';
 
 export function Home() {
-  return (
-    <HausLayout navLinks={[{ label: 'NavLink', href: '/' }]}>Hi!</HausLayout>
-  );
+  useEffect(() => {
```
Comment (jordanlesich) [inline apps/core-app/src/pages/Home.tsx:5]: useLayoutEffect? so we don't have to load this page first

File: apps/core-app/src/pages/Member.tsx [hunk 4]
```diff
@@ -21,74 +40,228 @@ const OverviewCard = styled(Card)`
...
+    balance: string;
+    fiatBalance: string;
+  };
+
+  const tableData: TokenTableType[] | null = useMemo(() => {
+    if (dao && currentMember) {
+      return dao.tokenBalances
+        .filter((bal) => Number(bal.balance))
+        .map((bal) => {
+          console.log('bal', bal);
```
Comment (jordanlesich) [inline apps/core-app/src/pages/Member.tsx:144]: clean

File: apps/core-app/src/utils/dataFetchHelpers.ts [hunk 5]
```diff
@@ -80,16 +80,16 @@ export const loadUserMembership = async ({
...
 
     if (memberRes?.data?.member && shouldUpdate) {
-      setUserMembership(memberRes.data.member);
+      setMember(memberRes.data.member);
     } else {
-      setUserMembership(undefined);
+      setMember(undefined);
     }
   } catch (error) {
     console.error(error);
```
Comment (jordanlesich) [inline apps/core-app/src/utils/dataFetchHelpers.ts:88]: I guess should update goes here too. I'll need to start adding those to my catch statements as well

**Review by jordanlesich — APPROVED** [review]
Looking good. A few small notes.

**Pull request merged**